monitor.py

```diff
@@ -329,7 +329,16 @@ def poll_online_workers():
                     'Uploaded file should contain the string BYTES_IN_MB, contents:\n' + cat_result,
                 )
             uuid = run_command(
-                ['cl', 'run', 'stress-test.pl:' + upload_uuid, 'perl stress-test.pl 5 10 10']
+                [
+                    'cl',
+                    'run',
+                    '--request-memory',
+                    '10m',
```

Comment:
Could we factor this out into a separate variable (e.g., `... + small_memory + ...` where `small_memory = ['--request-memory', '10m']`?

Reply:
Not going to do that in this file, because it's only used once. Will refactor the way this works in `test_cli.py`, though.